sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -589,6 +621,11 @@ public void run() {
         }, updateCurrentActivityRunnable, updateCurrentActivityRunnable);
     }
 
+    @Override
+    protected int getTriggerInterval() {
```

Comment:
It returns an interval in millisecond. I wonder if we need to change the return type to long.

Reply:
Internally we use millis, so need to use long indeed